### DISCUSSION TITLE
Treatment for Prostate cancer

### DISCUSSION THREAD

**Discussion by ofajardo** [synthetichealth/synthea#1496] [Q&A] [answered]
Thanks for the Prostate cancer module, which looks very interesting! 

The data already looks good for our purposes (developing a training/course) as it is, but we would like to understand a bit better, if possible, the reasons behind the decisions in the model, particularly the treatments administered to the patients. 

What we observe in the data is consistent with the model: 

After the diagnosis of Neoplasm of Prostate is given:,
1- a small percentage of patients (~8% in my sample, 9% in the model) are diagnosed with metastatic prostate cancer. Those patients do not receive any surgical or pharmacological treatment but just go to watchful waiting and survive up to 5 years (median = 3 years).
2- A larger percentage of patients (~92% in my sample, 91% in the model) are diagnosed with In situ prostate cancer. Those patients get one initial cycle of chemotherapy + hormone therapy and surgery (prostatectomy). 
 2.1 Most of patients get after 3 months a second cycle of chemotherapy + hormone therapy and survive very long (median survival time >10 years). 
 2.2 Some fraction do not get a second cycle of pharmacotherapy (~6% in my sample, 5% in the model?), do also not progress to metastasis and survive rather short (up to 5 years, median = 2.5 years). 
 2.3 Some fraction (~10% in my sample ... in the model 9% of 91-5=86%?) progress from in-situ to metastatic cancer in 3 months, do not get any more therapy consistent with the other metastatic patients and survive similarly as the metastatic patients. 

The percentages in my sample are slightly different from the model, probably because I am restricting my analysis to patients with diagnosis within the last 10 years only, discarding the rest. 

We have some naive questions (as our knowledge in prostate cancer is basic) regarding this: 
* We have read in the literature that for in situ prostate cancer the therapies can be do nothing, or watchful waiting, surgery (prostatectomy), castration, radiotherapy etc but typically not chemotherapy; while in advanced prostate cancer such as metastatic prostate cancer and castration resistant prostate cancer (not modelled here if I am not mistaken) the treatment can include chemotherapy [1](https://pubmed.ncbi.nlm.nih.gov/34153403/)[2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4124639/), the same applies to veterans [3](https://www.cancer.va.gov/assets/pdf/clinical-pathways/13/PrCCP.pdf) [4](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4831912/). It is also reflected in the decision tree in the screenshot attached [source](https://pc-decisiontree.com/step-9-2/) . It looks like therefore as if the treatment options are inverted between in situ and metastatic prostate cancer. What is the reason behind this modelling decision? Any citations would also be appreciated. 
* Similarly what is the meaning of those patients who receive only one cycle of chemotherapy instead of two and survive shorter? 

We are not asking to modify the module, as we can use the data as is, but we ask, because we have the feeling we may be missing something in the treatment landscape!. 

sorry for the naive questions and thanks in advance!

<img width="973" alt="image" src="https://github.com/user-attachments/assets/48012ac0-c304-47f4-a050-cf630b17239d">

**Comment by jawalonoski** [comment, marked as answer]
@ofajardo The clinician who worked on the prostate cancer module developed the clinical content approximately 6 years ago, and does not currently work with our team. Unfortunately, the module notes/remarks are empty. The notes that I have indicate that, at the time, he worked with the Lead at the National Oncology Program Office in Specialty Care Services, Department of Veterans Affairs, who provided order sets, and treatment regimens for prostate, colorectal and lung cancers.

It is not specified what format or references this information was provided in. It is also possible, that treatments have changed over time or vary geographically. It is also possible our model is just wrong.

**Reply by ofajardo** [reply]
Thanks for the info!
